### ISSUE TITLE
Alex keeps on asking me if I have applied the code changes

### ISSUE THREAD

**Issue by owenzhao** [DanielEdrisian/AlexSideBar-Public#68]
**Describe the bug**
See the bottom of the screenshot. Alex keeps asking me that. When I choose either of the buttons, it will show the code again and then asks me again with the same question.

**Screenshots/Screen recordings**
<img width="858" alt="截屏2024-12-26 08 18 49" src="https://github.com/user-attachments/assets/dc488731-15b7-4d7a-a2a9-239a6c4c8772" />

**What version are you on currently?**
 - Version 2.0.0 (100)

**Comment by rudrankriyam** [comment]
So it is a loop asking the **same code** again and again?

**Comment by owenzhao** [comment]
> So it is a loop asking the **same code** again and again?

Yes. If I answer click either of the buttons. It will show the codes again and ask again. Further more, when I ask another question in the below textview, Alex will answer the question and ask the same question. I didn't click the new question as I knew it was unnecessary. I finally got rid of it by creating a new chat.

**Comment by owenzhao** [comment]
I think you should investigate why there is a question? What is the purpose of the question in Alex's source code?
